vital/tests/rpc_reader.h

```diff
@@ -44,6 +44,7 @@ kwiver::vital::simple_camera_rpc read_rpc( std::string filename )
   kwiver::vital::vector_3d world_offset;
   kwiver::vital::vector_2d image_scale;
   kwiver::vital::vector_2d image_offset;
+  kwiver::vital::vector_2i image_dimension;
```

Comment:
We should initialize this vector to (0,0) because existing rpc files will not have the additional line for width and height.

Reply:
done